spring-cloud-stream-module-launcher/src/main/java/org/springframework/cloud/stream/module/launcher/ModuleLauncherProperties.java

```diff
@@ -43,4 +71,16 @@ public void setModules(String[] modules) {
 		return modules;
 	}
 
+	@PostConstruct
+	public void deduceModuleArguments() throws Exception {
```

Comment:
I think this makes sense, but I'm afraid we may hit the "map with environment variables" problem (which ought to be fixed, IIUC)